### PR TITLE
Create Serbian README

### PR DESCRIPTION
**Pull request by nexiRS**
#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [x] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

Added translation of the README into Serbian.

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by TobiGr — CHANGES_REQUESTED** [review]
Thank you. 
Link the Serbian file in the other translated READMEs, too please.

**Comment by nexiRS** [comment]
> Thank you. Link the Serbian file in the other translated READMEs, too please.

You're welcome!

Note: I needed to make a separate commit for the all READMEs, I'm not that experienced on GitHub haha.

File: doc/README.sr.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+<h3 align="center"><html>Планирамо да <i>препишемо</i> велике делове кода, како бисмо створили <a href="https://github.com/TeamNewPipe/NewPipe/discussions/10118">нови, модеран и стабилан NewPipe</a>!</h3>
+<h4 align="center"><b>Немојте</b> сада слати захтеве за <i>нове функције</i>, прихватаће се само захтеви за исправљање грешака.</h4>
+
+<p align="center"><a href="https://newpipe.net"><img src="assets/new_pipe_icon_5.png" width="150"></a></p>
```
Comment (TobiGr) [inline doc/README.sr.md:4]: relative URL
```suggestion
<p align="center"><a href="https://newpipe.net"><img src="../assets/new_pipe_icon_5.png" width="150"></a></p> 
```

**Review by TobiGr — CHANGES_REQUESTED** [review]
Thanks, We will take care of the commits later when merging this PR. Many images have relative URLs which need to be fixed.

**Comment by nexiRS** [comment]
> Thanks, We will take care of the commits later when merging this PR. Many images have relative URLs which need to be fixed.

Done. Thanks for the help!

**Comment by nexiRS** [comment]
> thanks!

Hey, I just noticed that I didn't fix URLs for images in the "Donation" section. Forgot to replace with ../assets/. Fix that please, if that's possible

**Pull request merged**